charts/argocd-apps/values-test.yaml

```diff
@@ -295,3 +295,100 @@ applications:
         value: secret  # TODO: set SECRET_KEY_BASE and make it a secret
       - name: GOVUK_APP_NAME
         value: collections
+- name: whitehall-admin
+  helmValues:
+    appImage:
+      repository: 172025368201.dkr.ecr.eu-west-1.amazonaws.com/whitehall
+      tag: f01a5161aa0d93b2efdd0775c61081695f12447a  # To be edited by automation (not yet implemented).
+    dbMigrationEnabled: true
+    workerEnabled: true
+    ingress:
+      enabled: true
+      annotations:
+        alb.ingress.kubernetes.io/load-balancer-name: whitehall-admin
+      hosts:
+      - name: whitehall-admin.eks.test.govuk.digital
+    replicaCount: 1
+    workerReplicaCount: 1
+    extraEnv: &whitehall-envs
+      - name: GOVUK_APP_NAME
+        value: whitehall
+      - name: GOVUK_APP_TYPE
+        value: rack
+      - name: NODE_ENV
+        value: production
+      - name: RACK_ENV
+        value: production
+      - name: RAILS_ENV
+        value: production
+      - name: GDS_SSO_OAUTH_ID
```

Comment:
we should create a single secret for GDS_SSO_OAUTH with format:
```json
{
clientID: <secret>,
clientSecret: <secret>
}
```

Reply:
i can potentially look at this as  separate task

Reply:
There should already be such a secret because these are generated by Bill's Signon bootstrap rake task.

You should be able to just refer to it like this:

```
      - name: GDS_SSO_OAUTH_ID
        valueFrom:
          secretKeyRef:
            name: signon-app-whitehall-eks
            key: oauth_id
```

If you've created any of these GDS_SSO_* secrets manually, you'll want to delete them and then run Bill's Signon bootstrap job. It looks like it's already [configured](https://github.com/alphagov/govuk-helm-charts/blob/main/charts/signon-resources/templates/bootstrap-job.yaml) to create some Whitehall stuff.